oic.r.acl2.raml

```diff
@@ -20,6 +20,24 @@ traits:
   get:
     description: |
       Retrieves the ACL data.
+      When used without query parameters, all the ACE entries are returned.
+      When used with an aceid or resources query parameter, only the
+      ACEs matching the specified parameter are returned.
+    queryParameters:
+      aceid:
+          type: integer
+          description: |
+            Returns ACE with the specificed aceid
+          required: false
+          repeat: false
+          example: 2112
+      resources:
```

Comment:
Looking closer at the IoTivity code, "resources" is only processed in conjunction with a subject. So do we want to retain that pattern or does "resources" make sense by itself?